src/components/navbar/DropDown.tsx

```diff
@@ -1,44 +1,57 @@
 "use client";
 
+import "reflect-metadata";
 import { ChevronDownIcon } from "@heroicons/react/24/outline";
 import Link from "next/link";
+import { useRouter } from "next/navigation";
 import Button from "@/components/Button";
 import { useAppDispatch, useUser } from "@/store/hooks";
 import { clientSignOut } from "@/store/features/auth/authSlice";
-import { serverSignOut } from "@/app/(auth)/authService";
-import { onOpenModal } from "@/store/features/modal/modalSlice";
+import { TYPES } from "@/di/types";
+import { resolve } from "@/di/resolver";
+import { type AuthClientAdapter } from "@/modules/auth/adapters/primary/authClientAdapter";
+import routePaths from "@/utils/routePaths";
 
 export default function DropDown({ openState }: { openState?: boolean }) {
+  const router = useRouter();
   const dispatch = useAppDispatch();
   const allVoyages = useUser().voyageTeamMembers;
   const activeVoyage = allVoyages?.find(
     (item) => item.voyageTeam.voyage.status.name === "Active",
   );
 
-  const currentVoyage = activeVoyage?.voyageTeam.name
-    ? `Team - Tier ${activeVoyage.voyageTeam.name
+  let currentVoyage;
+
+  if (activeVoyage?.voyageTeam.name) {
+    currentVoyage = `Team - Tier ${activeVoyage.voyageTeam.name
       .split("-")[1]
       .split("tier")[1]
       .toUpperCase()} ${activeVoyage.voyageTeam.name
       .split("-")[0]
-      .toUpperCase()}`
-    : "Please join a voyage to see your status information.";
+      .toUpperCase()}`;
+  } else {
+    currentVoyage = "Please join a voyage to see your status information.";
+  }
   const closed = "hidden";
   const open =
     "absolute flex flex-col gap-5 z-[1] w-[250px] p-5 bottom-100 translate-y-[15%] shadow-md bg-base-200 right-0 border border-base-100 rounded-2xl";
 
+  // TODO: update error handling
   async function handleClick() {
-    const [res, error] = await serverSignOut();
+    const authAdapter = resolve<AuthClientAdapter>(TYPES.AuthClientAdapter);
```

Comment:
In my opinion this is not the proper way to use the classes inside the components, but if this is working now I will fix it in my next task

Reply:
Sure, I'd like to see what suggestion you have. Do you want to go ahead and merge this for now? This is how I did it with the login (which has been merged already)

Reply:
The fix is straightforward and is related to the task on ClickUp. If you agree, I prefer to merge this pr and then do the fix. If things work I would like to complete the migration server >>> client components and then do the task on ClickUp